kuma/settings/common.py

```diff
@@ -931,7 +926,19 @@ def _get_languages_and_locales():
         'output_filename': 'build/js/ace.js',
     },
 }
-
+PIPELINE = {
+    'STYLESHEETS': PIPELINE_CSS,
+    'JAVASCRIPT': PIPELINE_JS,
+    'DISABLE_WRAPPER': True,
+    'JS_COMPRESSOR': 'pipeline.compressors.uglifyjs.UglifyJSCompressor',
```

Comment:
If we load this from the environment as well, we might be able to merge before rolling uglify / yuglify / whatever to Vagrant and production.